### ISSUE TITLE
Code Table Request - vaginal perforation [RANGES project]

### ISSUE THREAD

**Issue by bryansmclean** [ArctosDB/arctos#6594]
## Goal

Accurately record information about traits of cataloged mammals. This request formalizes agreements by Arctos community members participating in RANGES https://github.com/ArctosDB/arctos/issues/5532. If the requested attribute is a mammal reproductive trait, some discussions began prior to RANGES https://github.com/ArctosDB/arctos/issues/5264.

## Proposed new value. 

vaginal perforation

## Proposed Definition: 

Degree of perforation of the female vagina from reproduction.

## Attribute data type: 

categorical

## Attribute controlled values: 

open, closed unplugged, closed plugged

## Context

No current attribute for openness of the female vagina exists. The requested attribute fixes this and has been agreed upon by Arctos members participating in RANGES.

## Table

https://arctos.database.museum/info/ctDocumentation.cfm?table=ctattribute_type

## Collection type

Mamm

## Priority

High

Example Data: Requests with clarifying sample data are generally much easier to understand and prioritize. Please attach or link to any representative data, in any form or format, which might help clarify the request.

N/A

## Available for Public View

YES

_Project:

Discussion: Please reach out to anyone who might be affected by this change. Leave a comment or add this to the Committee agenda if you believe more focused conversation is necessary.

https://github.com/orgs/ArctosDB/teams/arctos-code-table-administrators @dbloom

Approval
All of the following must be checked before this may proceed.

The [How-To Document](https://handbook.arctosdb.org/how_to/How-To-Manage-Code-Table-Requests.html) should be followed. Pay particular attention to terminology (with emphasis on consistency) and documentation (with emphasis on functionality). No person should act in multiple roles; the submitter cannot also serve as a Code Table Administrator, for example.

- [ ]  Code Table Administrator[1] - check and initial, comment, or thumbs-up to indicate that the request complies with the how-to documentation and has your approval
- [ ]  Code Table Administrator[2] - check and initial, comment, or thumbs-up to indicate that the request complies with the how-to documentation and has your approval
- [ ]  DBA - The request is functionally acceptable. The term is not a functional duplicate, and is compatible with existing data and code.
- [ ]  DBA - Appropriate code or handlers are in place as necessary. (ID_References, Media Relationships, Encumbrances, etc. require particular attention)
Rejection
If you believe this request should not proceed, explain why here. Suggest any changes that would make the change acceptable, alternate (usually existing) paths to the same goals, etc.

Can a suitable solution be found here? If not, proceed to (2)
Can a suitable solution be found by Code Table Committee discussion? If not, proceed to (3)
Take the discussion to a monthly Arctos Working Group meeting for final resolution.
Implementation
Once all of the Approval Checklist is appropriately checked and there are no Rejection comments, or in special circumstances by decree of the Arctos Working Group, the change may be made.

Review everything one last time. Ensure the How-To has been followed. Ensure all checks have been made by appropriate personnel.

Make changes as described above. Ensure the URL of this Issue is included in the definition.

Close this Issue.

DO NOT modify Arctos Authorities in any way before all points in this Issue have been fully addressed; data loss may result.

Special Exemptions
In very specific cases and by prior approval of The Committee, the approval process may be skipped, and implementation requirements may be slightly altered. Please note here if you are proceeding under one of these use cases.

Adding an existing term to additional collection types may proceed immediately and without discussion, but doing so may also subject users to future cleanup efforts. If time allows, please review the term and definition as part of this step.
The Committee may grant special access on particular tables to particular users. This should be exercised with great caution only after several smooth test cases, and generally limited to "taxonomy-like" data such as International Commission on Stratigraphy terminology.

**Comment by bryansmclean** [comment]
Also requesting creation of a code table for the above categories.

term | definition 
-- | -- 
open | vagina perforated
closed unplugged | vagina not perforated
closed plugged | vagina plugged with copulation/sperm plug or detritus

**Comment by jldunnum** [comment]
What about just having "closed" and plugged and unplugged if known as modifers somewhere. Otherwise we will also need just a "closed" along with the other two. Vast majority of our data do not reflect plugged or not perforated so we need the simple default of "closed".

**Comment by mkoo** [comment]
I like that:
open | closed (add whether unplugged or plugged and with what in methods or remarks)

**Comment by dustymc** [comment]
If this is a yes/no situation, can the term be phrased to use https://arctos.database.museum/info/ctDocumentation.cfm?table=ctyes_no instead of requiring a new CT?

**Comment by Jegelewicz** [comment]
vagina perforated - yes/no

**Comment by bryansmclean** [comment]
> vagina perforated - yes/no

I thought this might work, but "perforation = no" is not equivalent to "closed", since closed can mean multiple things as suggested in my code table request. 

In particular, in the case of a copulatory plug, perforation presumably occurred but has not been observed directly.

**Comment by bryansmclean** [comment]
> What about just having "closed" and plugged and unplugged if known as modifers somewhere. Otherwise we will also need just a "closed" along with the other two. Vast majority of our data do not reflect plugged or not perforated so we need the simple default of "closed".

Yes, but Id like for us to force future mammalogists to be more specific than the older data are....

**Comment by Jegelewicz** [comment]
vagina perforated yes/no
vagina closed yes/no
?

**Comment by bryansmclean** [comment]
In the case of legacy data, these are synonyms as @jldunnum noted.

**Comment by Jegelewicz** [comment]
> I'd like for us to force future mammalogists to be more specific than the older data are

Our code tables aren't going to do that - people do this before entering any data and they collect whatever the datasheet allows them to collect. Tradition and social norms for this are more important than whatever we happen to have in the code table. If the ASM said "appropriate data collection includes....", it would make perfect sense to add those attributes. For now it seems we are just working with a small subset of the larger community and trying not to overwhelm our limited resources.

**Comment by bryansmclean** [comment]
I see your point, but doesn't the prospect of changing traditions for the better go hand-in-hand with developing the data infrastructure to do that? Certainly we don't all want Arctos to be responsive *only* to the needs presented by legacy data. Legacy data are often pretty limited in scope for individual specimens - what if we only developed tools that allowed us to do better and better at capturing limited data? Im sure lots of us are thinking about current and future goals when we contribute ideas to development of Arctos. Arctos does lots of things that societies havent yet endorsed or comprehended. I think having the capability to do certain things in Arctos **first** can inform what traditions of data collection and curation we build going forward. I agree this is a social problem but the question is who exactly leads the change.

This is obviously way beyond our discussion of vaginal perforation...
